components/plan-build/bin/environment.bash

```diff
@@ -463,12 +452,7 @@ do_setup_environment_wrapper() {
     # runtime path onto the front of the system path
     for k in ${!__runtime_environment[@]}; do
         local v="${__runtime_environment[${k}]}"
-        if [ "${k}" == "PATH" ]; then
-            build_line "Value of ${k} is ${!k}"
-            export "${k}"="$(push_to_path ${v} ${!k})"
-        else
-            export "${k}"="${v}"
-        fi
+        export "${k}"="${v}"
```

Comment:
Is`__set_runtime_env_path` expected to run before or after `set_buildtime_env`? They operate on the same data structures, so it'd be nice to document the assumptions for the use of each.